packages/apps/composer-app/src/playwright/first-run.spec.ts

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { test } from '@playwright/test';
+import { expect } from 'chai';
+import waitForExpect from 'wait-for-expect';
+
+import { AppManager } from './app-manager';
+
+test.describe('First-run tests', () => {
+  let host: AppManager;
+
+  test.beforeEach(async ({ browser }) => {
+    host = new AppManager(browser, true);
+    await host.init({ dontDismissFirstRun: true });
+  });
+
+  test.afterEach(async () => {
+    await host.closePage();
+  });
+
+  test('help plugin tooltip displays (eventually) on first run and not after refresh', async () => {
```

Comment:
I think these probably need to be two different tests, no? In one you end up clicking the "finish" button to dismiss and in the other when you reload it's gone

Reply:
fair enough!